clock/testing/fake_clock.go

```diff
@@ -116,6 +117,25 @@ func (f *FakeClock) NewTimer(d time.Duration) clock.Timer {
 	return timer
 }
 
+// AfterFunc is the Fake version of time.AfterFunc(d, cb).
```

Comment:
Add a unit test for AfterFunc:

https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/util/clock/clock_test.go#L150

Reply:
Will do, thanks!

@wojtek-t do you know how the test failure can be rectified? The failure is occurring because the `Clock` interface is no longer compatible with the previous version. 

cc @liggitt